.tekton/pull-request.yaml

```diff
@@ -7,6 +7,7 @@ metadata:
     pipelinesascode.tekton.dev/on-event: "pull_request"
     pipelinesascode.tekton.dev/on-target-branch: "main"
     pipelinesascode.tekton.dev/task: "git-clone"
+    pipelinesascode.tekton.dev/task: "yaml-lint"
```

Comment:
to avoid issue with yaml validation, please use  `pipelinesascode.tekton.dev/task-2: "yaml-lint"`